cmd/snap/main.go

```diff
@@ -129,11 +129,19 @@ func init() {
 	}
 }
 
+var accessDeniedHelp = `Access denied.
+
+Please use "sudo" or the "snap login <userid>" command
```

Comment:
At the risk of bikeshedding, can we make it more evident that we recommend doing `snap login`? I'd go as far as not even mentioning sudo, but maybe that's just me. With sudo some things won't work properly *anyway*, as what's really needed is to be logged in.